backend/dataall/modules/dataset_sharing/api/mutations.py

```diff
@@ -67,11 +67,25 @@
 
 revokeItemsShareObject = gql.MutationField(
     name='revokeItemsShareObject',
-    args=[gql.Argument(name='input', type=gql.Ref('RevokeItemsInput'))],
+    args=[gql.Argument(name='input', type=gql.Ref('ShareItemSelectorInput'))],
     type=gql.Ref('ShareObject'),
     resolver=revoke_items_share_object,
 )
 
```

Comment:
Very minor: being a bit philosophical, `verifyItemsShareObject` is more of a Query than a Mutation no?

Reply:
I would argue that it is more of a Mutation since we are changing the data (i.e. changing the share item health statuses)

Reply:
ahhaahha okey, you convinced me :)